LoopKitUI/Views/Settings Editors/CorrectionRangeOverridesEditor.swift

```diff
@@ -85,7 +85,15 @@ public struct CorrectionRangeOverridesEditor: View {
     private var contentWithCancel: some View {
         content
             .navigationBarBackButtonHidden(shouldAddCancelButton)
-            .navigationBarItems(leading: leadingNavigationBarItem)
+            .navigationBarItems(leading: leadingNavigationBarItem, trailing: unsetButton)
+    }
+
+    private var unsetButton: some View {
+        Button( action: {
+                value.ranges = [:];
+                presetBeingEdited = nil
+        } )
+        { Text(LocalizedString("Unset", comment: "Restore settings to nil")) }
```

Comment:
Heh, that was my least favorite of the three choices.  I just included it for completeness, but now I'm thinking that having the word "Unset" as standalone text (albeit blue colored) might read more like an adjective, rather than a verb. I won't block it on that if you think this is the right choice for some reason, but I do think "Delete" or "Clear" better communicates that the text is an action to take.  Unset is also fairly rare as a verb, I think.

Reply:
Changed button name to Delete.

BTW - there is comment text associated with that button.  When is that used?  Is it part of accessibility access?
I modified the comment as well.

Reply:
It's used to give more information about the context of the string to translators.